tests/testthat/test-epinowcast.R

```diff
@@ -94,7 +94,42 @@ test_that("epinowcast can fit a simple reporting model", {
   expect_error(nowcast$fit[[1]]$summary("rep_beta"))
 })
 
-test_that("epinowcast can fit a reporting model with a day of the week random
+test_that("epinowcast() can fit a simple reporting model where the max delay is greater than the empirical max delay", {
+  skip_on_cran()
+  skip_on_local()
+
+  pobs_long_delay <- suppressWarnings(
+    pobs$obs[[1]][, .group := NULL] |>
+    enw_filter_reference_dates(include_days = 20) |>
+    enw_preprocess_data(max_delay = 30)
+  )
+
+  nowcast <- suppressMessages(epinowcast(pobs_long_delay,
+    fit = enw_fit_opts(
+      sampler = silent_enw_sample,
+      save_warmup = FALSE, pp = FALSE,
+      chains = 2, iter_warmup = 250, iter_sampling = 500,
+      refresh = 0, show_messages = FALSE
+    ),
```

Comment:
I get a few divergent transitions with this example, but I guess in part this is also because we use a short warmup here...

Reply:
Yeah it really isn't ideal. I've opened an issue to go through and try and identify performance bottlenecks with the examples #350 which hopefully should help improve this.